pkg/api/utils.go

```diff
@@ -7,26 +7,12 @@ import (
 	"net/mail"
 
 	"github.com/grafana/grafana/pkg/api/response"
-	"github.com/grafana/grafana/pkg/middleware/cookies"
 	contextmodel "github.com/grafana/grafana/pkg/services/contexthandler/model"
 	"github.com/grafana/grafana/pkg/services/login"
 	"github.com/grafana/grafana/pkg/services/user"
 	"go.opentelemetry.io/otel/trace"
 )
 
-func (hs *HTTPServer) GetRedirectURL(c *contextmodel.ReqContext) string {
```

Comment:
This function is used in Grafana Enterprise. [1](https://github.com/grafana/grafana-enterprise/blob/d9f6e949c12978d09b16a64fef89cb904db4e57e/src/pkg/extensions/saml/api.go#L132) [2](https://github.com/grafana/grafana-enterprise/blob/d9f6e949c12978d09b16a64fef89cb904db4e57e/src/pkg/extensions/saml/api.go#L257)

I couldn't find any open branches in Grafana Enterprise for this.

Reply:
Found it! https://github.com/grafana/grafana-enterprise/pull/7141